shardingsphere-sql-parser/shardingsphere-sql-parser-binder/src/main/java/org/apache/shardingsphere/sql/parser/binder/SQLStatementContextFactory.java

```diff
@@ -110,6 +112,9 @@ private static SQLStatementContext getDMLStatementContext(final SchemaMetaData s
         if (sqlStatement instanceof InsertStatement) {
             return new InsertStatementContext(schemaMetaData, parameters, (InsertStatement) sqlStatement);
         }
+        if (sqlStatement instanceof CallStatement) {
```

Comment:
Hi `CallStatement` belongs to `DMLStatement`, doesn't it?

Reply:
> CallStatement

Yes, `CallStatement` belongs to `DMLStatement`.